ext/standard/basic_functions.c

```diff
@@ -1237,6 +1238,11 @@ PHP_FUNCTION(time_sleep_until)
 		RETURN_FALSE;
 	}
 
+	if (UNEXPECTED(!(target_secs >= 0 && target_secs <= top_target_sec))) {
+		zend_argument_value_error(1, "must be at most " ZEND_LONG_FMT, (uint64_t)top_target_sec);
```

Comment:
The error message wording is no longer right now, should be a "between"-style wording.